packages/itmat-interface/test/serverTests/users.test.ts

```diff
@@ -420,24 +436,29 @@ describe('USERS API', () => {
             });
 
             /* getting the id of the created user from mongo */
-            const createdUserId = (await mongoClient
+            // const createdUserId = (await mongoClient
+            //     .collection(config.database.collections.users_collection)
+            //     .findOne({ username: 'testuser1' }, { projection: { id: 1 } })).id;
+
```

Comment:
Should the comment block still be present ?

Reply:
done